x/ccv/provider/types/genesis_test.go

```diff
@@ -39,6 +39,7 @@ func TestValidateGenesisState(t *testing.T) {
 				nil,
 				nil,
 				nil,
+				nil,
```

Comment:
Would a test make sense here on invalid `lastProviderConsensusValidators `?
Wouldn't a `nil` provider consensus val set be invalid (?) Could that ever be possible?

Reply:
it was removed because the parameter was unused, anyways.
the set is computed on-the-fly